chemistrylab/extract_bench/extraction_0.py

```diff
@@ -33,14 +33,14 @@
 EthylAcetate = material.EthylAcetate
```

Comment:
Would it make more sense to make this an object rather than doing it multiple times later?